core/domain/prod_validators.py

```diff
@@ -3151,6 +3152,182 @@ def _get_model_domain_object_instance(cls, item):
+            expected_translation_suggestion_count = (
+                suggestion_models.GeneralSuggestionModel.get_all()
+                .filter(
+                    suggestion_models.GeneralSuggestionModel.status == (
+                        suggestion_models.STATUS_IN_REVIEW))
+                .filter(
+                    (
+                        suggestion_models.GeneralSuggestionModel
+                        .suggestion_type
+                    ) == suggestion_models.SUGGESTION_TYPE_TRANSLATE_CONTENT)
+                .filter(
+                    (
+                        suggestion_models.GeneralSuggestionModel
+                        .language_code
+                    ) == language_code)
+                .count()
+            )
+            if language_code in item.translation_suggestion_counts_by_lang_code:
+                model_translation_suggestion_count = (
+                    item.translation_suggestion_counts_by_lang_code[
+                        language_code]
+                )
+                if model_translation_suggestion_count != (
+                        expected_translation_suggestion_count):
+                    cls._add_error(
+                        'translation suggestion %s' % (
+                            base_model_validators.ERROR_CATEGORY_COUNT_CHECK),
+                        'Entity id %s: Translation suggestion count for '
+                        'language code %s: %s does not match the expected '
+                        'translation suggestion count for language code %s: '
+                        '%s' % (
+                            item.id, language_code,
+                            model_translation_suggestion_count, language_code,
+                            expected_translation_suggestion_count)
+                        )
+            elif expected_translation_suggestion_count != 0:
+                cls._add_error(
+                    'translation suggestion count %s' % (
+                        base_model_validators.ERROR_CATEGORY_FIELD_CHECK),
+                    'Entity id %s: The translation suggestion count for '
+                    'language code %s is %s, expected model '
+                    'CommunityContributionStatsModel to have the language code '
+                    '%s in its translation suggestion counts but it doesn\'t '
+                    'exist.' % (
+                        item.id, language_code,
+                        expected_translation_suggestion_count, language_code)
+                )
+
+    @classmethod
+    def _validate_question_reviewer_count(cls, item):
+        """Validate that the question reviewer count matches the number of
+        question reviewers in the datastore.
+
+        Args:
+            item: ndb.Model. CommunityContributionStatsModel to validate.
+        """
+        expected_question_reviewer_count = (
+            user_models.UserContributionRightsModel.query(
+                ( # pylint: disable=singleton-comparison
```

Comment:
Instead of using a pragma, can we just create a simple classmethod for this query?

Reply:
Ignore this comment, I miss interpreted the usage of pragma here!

Reply:
What do you mean? Whenever we filter or query we need to use the == sign which we need to disable pylint for correct?